packages/manager/apps/catalog-revamp/src/components/SearchBar/SearchBar.scss

```diff
@@ -0,0 +1,27 @@
+.searchbar-container {
+  margin: 20px;
+  float: right;
+
+  .searchbar {
+    float: right;
+
+    .search-button {
+      position: absolute;
+      margin-left: -10px;
+    }
+
+    osds-button {
+      border-radius: 0;
+      button {
+        border-radius: 0;
+        border-top-left-radius: 0px;
+        border-bottom-left-radius: 0px;
+      }
+    }
+
+    osds-input {
```

Comment:
We don't have to override the osds components css rules

Reply:
Without the width 250px the search bar is very small and don't really match the design
![image](https://github.com/ovh/manager/assets/33665069/1f9c9866-50cf-42b9-991a-7dcab6c70b88)

Reply:
Yes you can wrap in a div and pass the class inside it

Reply:
???

Reply:
Not working using class